src/os_posix/os_time.c

```diff
@@ -12,26 +12,26 @@
  * __wt_epoch --
  *	Return the time since the Epoch.
  */
-int
+void
 __wt_epoch(WT_SESSION_IMPL *session, struct timespec *tsp)
 {
 	WT_DECL_RET;
 
 #if defined(HAVE_CLOCK_GETTIME)
 	WT_SYSCALL(clock_gettime(CLOCK_REALTIME, tsp), ret);
 	if (ret == 0)
-		return (0);
-	WT_RET_MSG(session, ret, "clock_gettime");
+		return;
+	WT_PANIC_MSG(session, ret, "clock_gettime");
```

Comment:
I think the panic is okay given that we really have no expectation that it will fail.  I think the risk of a panic is less than the impact of, say, dealing with additional zeroed out time conditionals in the WT_TIMEDIFF macros and other places every time they're used.